src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceMetricsTestCase.java

```diff
@@ -0,0 +1,73 @@
+package org.jboss.hal.testsuite.test.runtime;
+
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.testsuite.fragment.MetricsAreaFragment;
+import org.jboss.hal.testsuite.fragment.MetricsFragment;
+import org.jboss.hal.testsuite.page.runtime.DataSourcesMetricsPage;
+import org.jboss.hal.testsuite.test.category.Shared;
+import org.jboss.hal.testsuite.util.Console;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author mkrajcov <mkrajcov@redhat.com>
+ */
+@RunWith(Arquillian.class)
+@Category(Shared.class)
+public class DataSourceMetricsTestCase {
+
+    public static final String AVAILABLE_CONNECTIONS = "Available Connections";
+    public static final String ACTIVE = "Active";
+    public static final String MAX_USED = "Max Used";
+    public static final String HIT_COUNT = "Hit Count";
+    public static final String ACCESS_COUNT = "Access Count";
+    public static final String MISS_COUNT = "Miss Count";
+    public static final int DELTA = 3;
+    @Drone
+    private WebDriver browser;
+
+    @Page
+    private DataSourcesMetricsPage dsPage;
+
+    @Before
+    public void before(){
+        browser.navigate().refresh();
+        Graphene.goTo(DataSourcesMetricsPage.class);
+        browser.manage().window().maximize();
```

Comment:
Did you consider to use org.jboss.hal.testsuite.util.Console.maximizeWindow() instead?